File: backend/cleanup.js
const fs = require('fs');
const path = require('path');

const backendDir = 'C:\\Users\\USER\\Desktop\\Smart-Campus\\backend';

const filesToRemove = [
    'Dockerfile',
    'docker-compose.yml',
    'compose-up.ps1',
    'github-workflow.yml',
    'mvnw',
    'mvnw.cmd',
    '.env.example',
    '.gitattributes'
];

const dirsToRemove = ['.mvn', 'target'];

console.log('🗑️  Starting cleanup...\n');

// Remove files
filesToRemove.forEach(file => {
    const filePath = path.join(backendDir, file);
    if (fs.existsSync(filePath)) {
        fs.unlinkSync(filePath);
        console.log(`✅ Removed: ${file}`);
    } else {
        console.log(`⏭️  Not found: ${file}`);
    }
});

// Remove directories
dirsToRemove.forEach(dir => {
    const dirPath = path.join(backendDir, dir);
    if (fs.existsSync(dirPath)) {
        fs.rmSync(dirPath, { recursive: true, force: true });
        console.log(`✅ Removed folder: ${dir}`);
    } else {
        console.log(`⏭️  Not found: ${dir}`);
    }
});

console.log('\n🎉 Cleanup complete!\n');
console.log('📁 Remaining files:');
fs.readdirSync(backendDir).filter(item => {
    return fs.statSync(path.join(backendDir, item)).isFile();
}).sort().forEach(file => {
    console.log(`   - ${file}`);
});

File: backend/cleanup.py
import os
import shutil

backend_dir = r'C:\Users\USER\Desktop\Smart-Campus\backend'

# Files to remove
files_to_remove = [
    'Dockerfile',
    'docker-compose.yml',
    'compose-up.ps1',
    'github-workflow.yml',
    'mvnw',
    'mvnw.cmd',
    '.env.example',
    '.gitattributes'
]

# Directories to remove
dirs_to_remove = ['.mvn', 'target']

print("🗑️  Starting cleanup...")
print()

# Remove files
for file in files_to_remove:
    file_path = os.path.join(backend_dir, file)
    if os.path.exists(file_path):
        os.remove(file_path)
        print(f"✅ Removed: {file}")
    else:
        print(f"⏭️  Not found: {file}")

# Remove directories
for dir_name in dirs_to_remove:
    dir_path = os.path.join(backend_dir, dir_name)
    if os.path.exists(dir_path):
        shutil.rmtree(dir_path)
        print(f"✅ Removed folder: {dir_name}")
    else:
        print(f"⏭️  Not found: {dir_name}")

print()
print("🎉 Cleanup complete!")
print()
print("📁 Remaining files:")
for item in sorted(os.listdir(backend_dir)):
    if os.path.isfile(os.path.join(backend_dir, item)):
        print(f"   - {item}")
